lms/djangoapps/instructor/enrollment.py

```diff
@@ -267,6 +269,15 @@ def reset_student_attempts(course_id, student, module_state_key, requesting_user
 
     if delete_module:
         module_to_reset.delete()
+        if block and block.has_score:
```

Comment:
So as @nasthagiri pointed out, some blocks not in the modulestore may still have their score reset (if content was changed or deleted, for example). How do we handle that case? Where does the max score come from?